### PR TITLE
add gitpod bot

### PR DESCRIPTION
**Pull request by akosyakov**

### CODE REVIEW

**Comment by akosyakov** [comment]
@svenefftinge could you review the content please?

File: _apps/gitpod.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Gitpod Bot
+description: Open issues and pull requests in Gitpod
+slug: gitpod-bot
+screenshots:
+- https://somewebsite.com/your-screenshot.png
```
Comment (akosyakov) [inline _apps/gitpod.md]: TODO: update screenshots

File: _apps/gitpod.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+authors: [ gitpod-io ]
+repository: gitpod-io/gitpod-bot
+host: https://gitpod-bot.now.sh
+---
+
+Open issues and pull requests in [Gitpod](https://medium.com/gitpod/gitpod-gitpod-online-ide-for-github-6296b907a886).
```
Comment (svenefftinge) [inline _apps/gitpod.md]: please link to www.gitpod.io

**Comment by akosyakov** [comment]
I've tried to fix build issues related to the gitpod bot, remaining don't seem to be related.

**Comment by akosyakov** [comment]
Hi, let's know if something else should be done.

**Pull request merged**